src/indexdata.py

```diff
@@ -476,101 +473,134 @@ def tosolr(self):
-                mydict['mmd_geographic_extent_rectangle_south'] = float(
+                mydict['geographic_extent_rectangle_south'] = float(
                     self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['mmd:south']),
-                mydict['mmd_geographic_extent_rectangle_east'] = float(
+                mydict['geographic_extent_rectangle_east'] = float(
                     self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['mmd:east']),
-                mydict['mmd_geographic_extent_rectangle_west'] = float(
+                mydict['geographic_extent_rectangle_west'] = float(
                     self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['mmd:west']),
+                mydict['geographic_extent_rectangle_srsName'] = self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['@srsName'],
                 mydict['bbox'] = "ENVELOPE("+self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['mmd:west']+","+self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['mmd:east']+","+ self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['mmd:north']+","+ self.mydoc['mmd:mmd']['mmd:geographic_extent']['mmd:rectangle']['mmd:south']+")"
 
+        self.logger.info('Add location element?')
+
+        """ Dataset production status """
+        if 'mmd:dataset_production_status' in self.mydoc['mmd:mmd']:
+            if isinstance(self.mydoc['mmd:mmd']['mmd:dataset_production_status'],
+                    dict):
+                mydict['dataset_production_status'] = self.mydoc['mmd:mmd']['mmd:dataset_production_status']['#text']
+            else:
+                mydict['dataset_production_status'] = str(self.mydoc['mmd:mmd']['mmd:dataset_production_status'])
+
+        """ Dataset language """
+        if 'mmd:dataset_language' in self.mydoc['mmd:mmd']:
+            mydict['dataset_language'] = str(self.mydoc['mmd:mmd']['mmd:dataset_language'])
+
+        """ Operational status """
+        if 'mmd:operational_status' in self.mydoc['mmd:mmd']:
+            mydict['operational_status'] = str(self.mydoc['mmd:mmd']['mmd:operational_status'])
+
+        """ Access constraints """
+        if 'mmd:access_constraint' in self.mydoc['mmd:mmd']:
+            mydict['access_constraint'] = str(self.mydoc['mmd:mmd']['mmd:access_constraint'])
+
+        """ Use constraint """
+        if 'mmd:use_constraint' in self.mydoc['mmd:mmd']:
+            mydict['use_constraint'] = str(self.mydoc['mmd:mmd']['mmd:use_constraint'])
+
+        """ Personnel """
+
+        if 'mmd:personnel' in self.mydoc['mmd:mmd']:
+            personnel_elements = self.mydoc['mmd:mmd']['mmd:personnel']
+
+            if isinstance(personnel_elements, dict): #Only one element
+                personnel_elements = [personnel_elements] # make it an iterable list
+
+            for personnel in personnel_elements:
```

Comment:
I do not understand the LUT here as I can't see these in the MMD spec. Need some explanation here to understand probably.